transform/mattermost-analytics/models/marts/product/features/_product__features__models.yml

```diff
@@ -66,19 +75,14 @@ models:
           The number of unique users captured the current day and past 29 days performing an action that cannot (yet) 
           be mapped to a feature.
 
-      - name: count_known_features_events_daily
```

Comment:
I believe these need to be added to the new file as well (referenced by tests)

Reply:
The have been readded.